superset/viz.py

```diff
@@ -333,6 +337,7 @@ def query_obj(self):
         }
 
         d = {
+            'timezone': client_tz,
```

Comment:
where this timzone property is used.
can you share a flow of method invocations that is stitching this e2e data flow